qanta/wikipedia/cached_wikipedia.py

```diff
@@ -149,7 +151,28 @@ def load_page(self, key: str):
             return CachedWikipedia.load_page(key)
         return raw
 
+    def get_wikidata(self, key: str):
+        web_page = "https://www.wikidata.org/w/api.php?action=wbsearchentities&search=" + key + "&language=en&format=json"
+        headers = {"Accept": "application/json"}
+        req = requests.get(web_page, headers=headers)
+
+        try:
+            if not req.json()['search']:
+                print('No results ' + search_term + " : " + search_term)
+            else:
+                id = req.json()['search'][0]['id']
+                url = req.json()['search'][0]['concepturi']
+                headers = {"Accept": "application/json"}
+                req = requests.get(url, headers=headers)
+                claims = req.json()['entities'][id]['claims']
+                return claims
+
+        except ValueError:
```

Comment:
Are there other errors to catch besides key errors? A few that can come to mind:
* Some sort of network error
* The wikidata API rate limits us, so we should wait for X seconds before trying again (look at wiki code for an example)